### PR TITLE
CMD Palette + Launcher tweaks

### PR DESCRIPTION
**Pull request by jasonLaster**
* keyboard accessible
* hover states for launcher
* quick open is narrower

### CODE REVIEW

File: src/ui/components/CommandPalette/CommandPalette.tsx [hunk 1]
```diff
@@ -32,13 +32,23 @@ const COMMANDS: Command[] = [
...
 function CommandPalette({ hideCommandPalette }: PropsFromRedux) {
   const [searchString, setSearchString] = useState("");
+  const [selectedIndex, setSelectedIndex] = useState(0);
 
   const onChange = (e: ChangeEvent<HTMLInputElement>) => {
     setSearchString(e.target.value);
   };
-  const onKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+
+  const onKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
```
Comment (abstractalgo) [inline src/ui/components/CommandPalette/CommandPalette.tsx:41]: Doesn't respond to "Enter/Return" key.

File: src/ui/components/CommandPalette/CommandButton.tsx [hunk 2]
```diff
@@ -4,22 +4,28 @@ import { connect, ConnectedProps } from "react-redux";
 import { actions } from "ui/actions";
 import { Command } from "./CommandPalette";
 
-type CommandButtonProps = PropsFromRedux & { command: Command };
+type CommandButtonProps = PropsFromRedux & { command: Command; selected: boolean };
```
Comment (abstractalgo) [inline src/ui/components/CommandPalette/CommandButton.tsx:7]: Usually, this is more appropriately named as `active`.
(sorry if this way too nitpicky)

Comment (jasonLaster) [reply src/ui/components/CommandPalette/CommandButton.tsx:7]: Nope - I like it

File: src/ui/components/CommandPalette/CommandPalette.tsx (hunk 1 above)
Comment (jasonLaster) [reply src/ui/components/CommandPalette/CommandPalette.tsx:41]: haha i was being lazy. just added it

**Pull request merged**